tests/lib/prepare-restore.sh

```diff
@@ -605,6 +605,22 @@ prepare_suite() {
 prepare_suite_each() {
     local variant="$1"
 
+    # Create runtime files in case those don't exist
+    # This is for the first test of the suite, can't be added to the prepare_suite
+    # because some suites could not be preparing the suite such as the tools suite
```

Comment:
assuming that I didn't misunderstood, I'd rephrase this to:

> This is for the first test of the suite. We cannot perform these operations in `prepare_suite` because not all suites are triggering it (for example the `tools` suite doesn't).

Reply:
done